### ISSUE TITLE
Evaluate changing the neighborhood field for a Google Map Pin

### ISSUE THREAD

**Issue by camposcristianeze** [penguin-academy/covid-19-py#88]
Neighbourhoods in Paraguay are quite confusing and not very well known by everyone. If users can just leave a pin on the map with a radio it could be easier to track the area and future cases. Here's an example of confusing neighborhoods:

![image](https://user-images.githubusercontent.com/1928706/77218157-133e1400-6b07-11ea-8752-909aa0a0700b.png)

**Comment by pA1nD** [comment]
ohh.. that's indeed a problem. @Deeliapatricia @camposcristianeze would it be better to search by the street?

**Comment by camposcristianeze** [comment]
Yep, def the idea won't be to use their exact address 👍 Streets are also repeating a bit so I'd avoid that one.

One thing I found is https://apps.senatics.gov.py/dinacopa/#/zona/zonaspostales 
Debugging this I got a JSON of all districts (which is maybe a more common thing to use here) and then neighborhoods... Thinking to make a component which detects closer districts by location and then filters the list of neighbourhoods? 

BTW How are you geo-filtering Paraguay using the `PlacesAutoComplete` component? Is it through an env var? Couldn't find it

**Comment by jmayalag** [comment]
How accurate is if the location provided by the browser? Could we add a `CovPy wants to know your location`?

**Comment by pA1nD** [comment]
@jmayalag I think that would be something that could scare a numbers of users away.

@camposcristianeze SelectPlace component, line 22.

```
.getPlacePredictions(
      {
        input: input,
        types: ['(regions)'],
        componentRestrictions: { country: 'py' }
      },
```
